docs/pages/2020_BIGMAP/sections/basics.rst

```diff
@@ -3,3 +3,658 @@
+* Run and monitor the status of processes.
+* Explore and visualize the provenance graph.
+
+.. _BIGMAP_2020_Basics:provenance:
+
+Provenance
+==========
+
+Before we dive in, we need to briefly introduce one of the most important concepts for AiiDA: *provenance*.
+An AiiDA database does not only contain the results of your calculations, but also their inputs and each step that was executed to obtain them.
+All of this information is stored in the form of a *directed acyclic graph* (DAG).
+As an example, :numref:`BIGMAP_2020_Basics_fig_intro_workchain_graph` shows the provenance of the calculations of this tutorial.
+
+.. _BIGMAP_2020_Basics_fig_intro_workchain_graph:
+.. figure:: include/images/workchain_graph.png
+    :scale: 30
+    :align: center
+
+    Provenance Graph of a basic AiiDA WorkChain.
+
+In the provenance graph, you can see different types of *nodes* represented by different shapes.
+The green ellipses are ``Data`` nodes, the blue ellipse is a ``Code`` node, and the rectangles represent *processes*, i.e. the calculations performed in your *workflow*.
+
+The provenance graph allows us to not only see what data we have, but also how it was produced.
+During this tutorial we will be using AiiDA to generate the provenance graph in :numref:`fig_intro_workchain_graph` step by step.
+
+.. note::
+
+  Throughout this tutorial we will be using the ``verdi`` command line interface.
+  Here's couple of tricks that will make your life easier:
+
+  * The ``verdi`` command supports **tab-completion**:
+    In the terminal, type ``verdi``, followed by a space and press the 'Tab' key twice to show a list of all the available sub commands.
+  * For help on ``verdi`` or any of its subcommands, simply append the ``--help/-h`` flag:
+
+    .. code-block:: console
+
+        $ verdi -h
+
+.. _BIGMAP_2020_Basics:data_nodes:
+
+Data nodes
+==========
+
+Before running any calculations, let's create and store a *data node*.
+AiiDA ships with an interactive IPython shell that has many basic AiiDA classes pre-loaded.
+To start the IPython shell, simply type in the terminal:
+
+.. code-block:: console
+
+    $ verdi shell
+
+AiiDA implements data node types for the most common types of data (int, float, str, etc.), which you can extend with your own (composite) data node types if needed.
+For this tutorial, we'll keep it very simple, and start by initializing an ``Int`` node and assigning it to the `node` variable:
+
+.. code-block:: ipython
+
+    In [1]: node = Int(2)
+
+We can check the contents of the ``node`` variable like this:
+
+.. code-block:: ipython
+
+    In [2]: node
+    Out[2]: <Int: uuid: eac48d2b-ae20-438b-aeab-2d02b69eb6a8 (unstored) value: 2>
+
+Quite a bit of information on our freshly created node is returned:
+
+* The data node is of the type ``Int``
+* The node has the *universally unique identifier* (**UUID**) ``eac48d2b-ae20-438b-aeab-2d02b69eb6a8``
```

Comment:
I would maybe be more explicit here that the UUID will be different for the user, to prevent any kind of pause at this point. Something like:

```suggestion
* The node has the *universally unique identifier* (**UUID**), which will be different in each case (in the example above, it turned out to be ``eac48d2b-ae20-438b-aeab-2d02b69eb6a8``)
```